src/main/webapp/app/overview/course-conversations/dialogs/conversation-detail-dialog/tabs/conversation-info/conversation-info.component.scss

```diff
@@ -4,8 +4,11 @@
             padding: 0.5rem;
 
             .action-button {
-                visibility: hidden;
                 margin: 0.25rem;
+                text-decoration: none;
+                &:hover {
```

Comment:
Codacy wants an empty line here

Reply:
Fixed in ea3ccf193077acfd40ee90e1f84b9fc2b1a43e6c